common/app/views/fragments/analytics/comscore.scala.html

```diff
@@ -6,7 +6,7 @@
 @if(ComscoreSwitch.isSwitchedOn) {
     <noscript>
         @defining(getContent(page).map(_.tags.keywords.map(_.name)).map(_.mkString(","))) { keywords =>
-            <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&comscorekw=@keywords.map({kw => URLEncoder.encode(kw, "UTF-8")})" />
+            <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&gdpr=0&comscorekw=@keywords.map({kw => URLEncoder.encode(kw, "UTF-8")})" />
```

Comment:
What does this do exactly?

Reply:
not sure, i've only updated the test checks